content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -0,0 +1,656 @@
+---
+id: get-started-with-the-rules-engine
+themes: boost-your-productivity, catalog-settings
+title: Get started with the **rules engine**
+popular: false
+ee-only: true
+related: what-is-a-rule
+---
+
+The rules engine can radically boost your productivity in the PIM by automating actions. The rules are defined in a YML file that you can import in the PIM, using a dedicated job profile.   
+
+This feature is very, like, really powerful, but it can also be a bit complex to apprehend. Reading this article, you will find out what you can use it for and how it works.
```

Comment:
I'd remove the "complex to apprehend" to turn it in a positive way.

Reply:
What do you propose?